### PR TITLE
locale: 'kr' is not language code

### PR DESCRIPTION
**Pull request by hiphapis**
'kr' is not language code. it's country code.
Korean language code is 'ko'
so i did rename to 'ko'

### CODE REVIEW

**Comment by acrobat** [comment]
To not break compatibility we should also keep the kr file, but add a deprecation notice at the top of the file (so we can remove it in 2.0)

Thanks!

File: js/locales/bootstrap-datepicker.ko.js [hunk 1]
```diff
@@ -3,11 +3,13 @@
...
-	$.fn.datepicker.dates['kr'] = {
+	$.fn.datepicker.dates['ko'] = {
 		days: ["일요일", "월요일", "화요일", "수요일", "목요일", "금요일", "토요일"],
 		daysShort: ["일", "월", "화", "수", "목", "금", "토"],
 		daysMin: ["일", "월", "화", "수", "목", "금", "토"],
 		months: ["1월", "2월", "3월", "4월", "5월", "6월", "7월", "8월", "9월", "10월", "11월", "12월"],
 		monthsShort: ["1월", "2월", "3월", "4월", "5월", "6월", "7월", "8월", "9월", "10월", "11월", "12월"]
 	};
+  // for compatibility
+  $.fn.datepicker.dates['kr'] = $.fn.datepicker.dates['ko'];
```
Comment (acrobat) [inline js/locales/bootstrap-datepicker.ko.js]: If the original ko file is restored you can remove this line

**Comment by hiphapis** [comment]
Great news!
yeah. i restored that line
Thanks ;)

**Comment by acrobat** [comment]
Great, please just restore the original ko file to (for people currently using the ko language code. But also keep the new kr language file just so that we have a alternative when we deleted in 2.0 (and add some deprecation comment in the old ko file)

**Comment by hemp** [comment]
#1367 spruced up support for ko/zh.

**Comment by hiphapis** [comment]
@acrobat Ah. you mean is restore legacy file. right?

**Comment by acrobat** [comment]
Thanks for the pr @hiphapis, but #1367  got merged. So closing this!

**Pull request closed without merging**